styles.css

```diff
@@ -8,5 +8,25 @@ body {
 }
 
 img {
-  width: 20%;
+  width: 8%;
+}
+
+.east {
+  transform: rotate(0turn);
+  transition: transform 0.5s;
```

Comment:
You should just put the `transition` on the `img` rule, so you don't have to repeat it four times